src/server/scripts/Northrend/Naxxramas/instance_naxxramas.cpp

```diff
@@ -94,7 +94,6 @@ class instance_naxxramas : public InstanceMapScript
             PatchwerkRoomTrash.clear();
 
             // Controls
-            _horsemanKilled = 0;
```

Comment:
implicit cast below, change to initialize to 0 instead
`_horsemanLoadDoneState = false;`